scripts/pr-merge-autopilot.sh

```
@@ -198,18 +198,18 @@ while true; do
     [[ -z "$name" ]] && continue
 
     case "$state" in
-      pass)
-        ((CHECKS_PASSED++))
-        status_green "$name"
+      pass|skipping)
+        CHECKS_PASSED=$((CHECKS_PASSED + 1))
+        [[ "$state" == "skipping" ]] && status_yellow "$name (skipped)" || status_green "$name"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)